### DISCUSSION TITLE
Automatic removal of previews in Vercel

### DISCUSSION THREAD

**Discussion by mbustosr** [vercel/community#2148] [Help] [answered]
### Summary

There is a method to automatically delete previews created or generated by Vercel in a certain period of time, for example, more than 5 days for better organization. Vercel does not have this option.

**Comment by swarnava** [comment, marked as answer]
I there. Indeed, we don't have a retention policy for preview deployment. You can write script using [Vercel API](https://vercel.com/docs/rest-api/endpoints#delete-a-deployment) to delete deployment as your desired preference. Thank you!

**Comment by mbustosr** [comment]
Okay, but what I need to know is if there is a method to want the automatic elimination of the preview environments that Vercel generates with each use. I have a GitLab integration with Vercel and when I create a MR it creates the preview, but when it is closed or merged with the main branch the preview continues and is not deleted.

I have a code to do it from GitLab but I don't know if the integration or GitLab through the code can eliminate these environments in Vercel.

This is possible? Because I have seen in various comments from the community that Vercel does not have this at the moment
